gradio/components/chatbot.py

```diff
@@ -88,14 +88,16 @@ class Message(GradioModel):
     metadata: Metadata = Field(default_factory=Metadata)
     content: Union[str, FileMessage, ComponentMessage]
 
+class ExampleMessage(TypedDict):
```

Comment:
@abidlabs thoughts on this Data structure for examples? If we want to support multimodal examples, that is essentially 2 Messages, the text and the file.

Reply:
I've seen multimodal chatbots where the examples are two images, etc. so I think for the general case, I would make the change below:

Reply:
Also, I looked at a few example chat applications like ChatGPT and HuggingChat, and its often the case that the display text is not the same as the message that is submitted to the chatbot. For example, you can see ChatGPT's starters look like this:

<img width="865" alt="image" src="https://github.com/user-attachments/assets/c2705573-63dd-4889-af01-346b9ea11742">

But if you click on them, you get much more text as the initial prompt.

I suggest adding an optional argument called `label` which takes in arbitrary Markdown and if it is provided, it is rendered in the starter bubble. If its not provided, then the text and files are rendered as you currently have